js_modules/dagster-ui/packages/ui-core/src/schedules/ScheduleSwitch.tsx

```diff
@@ -19,18 +19,18 @@ import {
 } from './types/ScheduleSwitch.types';
 import {usePermissionsForLocation} from '../app/Permissions';
 import {InstigationStatus} from '../graphql/types';
+import {INSTIGATION_STATE_BASE_FRAGMENT} from '../instigation/InstigationUtils';
 import {repoAddressToSelector} from '../workspace/repoAddressToSelector';
 import {RepoAddress} from '../workspace/types';
 
 interface Props {
   repoAddress: RepoAddress;
   schedule: ScheduleSwitchFragment;
   size?: 'small' | 'large';
-  shouldFetchLatestState?: boolean;
 }
 
 export const ScheduleSwitch = (props: Props) => {
-  const {repoAddress, schedule, size = 'large', shouldFetchLatestState} = props;
```

Comment:
Removed `shouldFetchLatestState` and instead just going to query it directly even if the parent component is passing an up to date schedule/sensor object because the risk of forgetting this is high but also we can now query individual instigation states efficiently.